src/opera/commands/deploy.py

```diff
@@ -1,12 +1,13 @@
 import argparse
+import typing
```

Comment:
I like typing. But do you think that this should break something?

Reply:
`typing` and the type hints I used are available since python 3.6 - and I've checked that the PyPI page lists Python 3.6 and 3.7, while tests use 3.6, 3.7 and 3.8. I can move this typing information to comments if you want to preserve Python 3.5 compatibility, if there is any.

Python 3.5 also [EOLs on 2020-09-13](https://devguide.python.org/#status-of-python-branches). :)

Reply:
Ok, then I think we are totally cool with that :)